app/models/video.rb

```diff
@@ -23,4 +24,11 @@ def display_large_video_image
     end
   end
   
+  # calulate average rating for video to 1 decimal point
+  def average_rating
+    sum_of_ratings = self.reviews.sum(:rating).to_f
```

Comment:
Thanks, I tried using average initially but i kept getting errors. No idea what I was doing wrong! That's it working now, much better!